### ISSUE TITLE
Stub editors draft pages and point to WAI APG site

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-practices#2420]
This issue is part of the [Restructure repository Project](https://github.com/w3c/aria-practices/projects/42), which includes sunsetting the editors draft.

Since the new WAI site can be updated weekly, the [editors draft of APG](http://w3c.github.io/aria-practices/) is no longer needed. Maintaining the editors draft creates unnecessary work and constrains site development because it requires maintaining the old respec TR document.

This issue tracks work needed to replace the editors draft with stub pages that refer readers to the new WAI APG site.

**Comment by mcking65** [comment]
This issue will be executed after #2417 and is merged, #2421 is complete, and the restructured content is published.
